transmitter.js

```diff
@@ -1,31 +1,33 @@
 let transmissionFailureCount = 0;
 
 function networkTransmitStub(celcius) {
-    console.log(`Temperature to transmit: ${celcius} celcius`);
+  console.log(`Temperature to transmit: ${celcius} celcius`);
 
-    // The intention here is to transmit the temperature over a network.
-    // However, this is a stub, so we just print the message above
-    // and give back a return code.
+  // The intention here is to transmit the temperature over a network.
+  // However, this is a stub, so we just print the message above
+  // and give back a return code.
 
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // This stub always succeeds and returns 200
+  // Return 200 for ok
+  // Return 500 for not-ok
+
+  //  failure when the temperature is above 100 celcius
+  if (celcius > 100) {
+    return 500;
+  } else {
     return 200;
+  }
 }
 
 function transmitInCelcius(farenheit) {

```

Comment:
`transmitInCelcius` is production code. However, this file includes the stub as well. That means consumers who need this functionality need to take the stub as well. Separate the stub into the `transmitter.test.js`.
Hint: If this results in circular dependency between files, lookup "dependency injection"